src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -2146,6 +2145,23 @@ bool AdaptDispatch::EnableAlternateScroll(const bool enabled)
     return success;
 }
 
+
+//Routine Description:
+// Enable "brackted paste mode".
+//Arguments:
+// - enabled - true to enable, false to disable.
+// Return value:
+// True if handled successfully. False otherwise.
+bool AdaptDispatch::EnableBracketedPasteMode(const bool /*enabled*/)
```

Comment:
I'm not sure what to do with conhost. Those legacy code always scare me . Call we just leave it like this?

Reply:
As long as it's not in the legacy section, you should be able to modify it with relative comfort. The legacy section was purposely created so that the rest of the code could be modernized. As long as there aren't any regressions, you should be fine to tweak this code as needed for supporting this feature.